src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -303,283 +425,95 @@ Entropy conserving two-point flux by
 """
 @inline function flux_chandrashekar(u_ll, u_rr, orientation, equations::CompressibleEulerMulticomponentEquations2D)
   # Unpack left and right state
-  @unpack gas_constant1, gas_constant2, cv1, cv2 = equations
-  rho1_ll, rho2_ll, rho_v1_ll, rho_v2_ll, rho_e_ll = u_ll
-  rho1_rr, rho2_rr, rho_v1_rr, rho_v2_rr, rho_e_rr = u_rr
+  @unpack gamma, gas_constant, cv = equations
+  rho_v1_ll, rho_v2_ll, rho_e_ll = u_ll
+  rho_v1_rr, rho_v2_rr, rho_e_rr = u_rr
+  rho_ll      = 0
+  rho_rr      = 0
+  rhok_mean   = SVector{length(gamma),Float64}(ln_mean(u_ll[i+3], u_rr[i+3]) for i = 1:length(gamma))
+  rhok_avg    = SVector{length(gamma),Float64}(0.5 * (u_ll[i+3] + u_rr[i+3]) for i = 1:length(gamma))
```

Comment:
You should try to avoid hard-coding `Float64`.

Reply:
done in 4c720b0